src/wallet/wallet.cpp

```diff
@@ -3968,6 +3969,64 @@ std::vector<std::string> CWallet::GetDestValues(const std::string& prefix) const
     return values;
 }
 
+bool CWallet::Verify(std::string wallet_file, bool salvage_wallet)
```

Comment:
Lock `cs_wallets`?

Reply:
nit, `const std::string& wallet_file`.

Reply:
> Lock cs_wallets?

Done. How does this interact with your WalletManager work? Was the idea that cs_wallets was only held by WalletManager?

Reply:
> nit, const std::string& wallet_file.

Thanks - fixed